### PR TITLE
Build UI and package info

### PR DESCRIPTION
**Pull request by rsvoboda**
Info to build UI and the package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+Overbård is a Kanban board integrating with Jira. It is implemented as a Jira plugin with an Angular 5 client running in the browser. It's evolution of Jirban project https://github.com/jirban/jirban-jira/
+
+## Build and test UI
+```bash
+cd webapp/
+node --version ## ensure you have version >=4 <=9
+yarn install && yarn build
+yarn start
+yarn test
+yarn test:travis-headless
```
[the remark this answers is not in the document]
Comment (rsvoboda) [reply README.md]: It's out there, yarn outdated showed already 6.0.0 :)

[the remark this answers is not in the document]
Comment (rsvoboda) [reply README.md]: https://github.com/angular/angular/releases 13 hours ...

[the remark this answers is not in the document]
Comment (rsvoboda) [reply README.md]: that was to angular ...

**Comment by kabir** [comment]
I think this is out of date

**Pull request closed without merging**